recipe/conda_build_config.yaml

```diff
@@ -1,8 +0,0 @@
-gdal:
-- 2.4
-libgdal:
-- 2.4
```

Comment:
@SrNetoChan do you know if the LTR builds against `gdal 3`? If not we would have to rebuild `gdal 2.4.x` against newer dependencies.

Reply:
3.4 should be kept at gdal 2.4.x It's not ready for gdal 3